merkle-js/src/generate_merkle_json.ts

```diff
@@ -0,0 +1,98 @@
+import { merkle, num,cairo } from 'starknet';
+import * as starkCurve from 'micro-starknet';
+import * as fs from 'fs';
+
```

Comment:
Can you please clean this file up and write instructions on how to use this clearly?